docs/marketplace-required-csv-annotations.md

```diff
@@ -5,25 +5,56 @@ An operator's CSV must contain the following annotations for it to be displayed
 ```yaml
 metadata:
   annotations:
-    categories: A list of comma separated categories that your operator falls under.
-    certified: The operator's certification. This field should be set to false until a process to get this value is defined at a later date.
-    description: |-
-      A short description of the operator that will be displayed on the marketplace tile.
-      If this annotation is not present, the `spec.description` value will be shown instead.
-      In either case, only the first 135 characters will appear.
+    capabilities: One of the following: Basic Install, Seamless Upgrades, Full Lifecycle, Deep Insights, Auto Pilot. For more information see https://www.operatorhub.io/images/capability-level-diagram.svg
+    categories: A comma separated list of categories from the values below. If not set, this will be set to "Other" in the UI
     containerImage: The repository that hosts the operator image. The format should match ${REGISTRYHOST}/${USERNAME}/${NAME}:${TAG}
     createdAt: The date that the operator was created. The format should match yyyy-mm-ddThh:mm:ssZ
-    support: The maintainer of the operator. This value should be a name, not an email.
+    support: The maintainer of the operator. This value should be a name, not an email
```

Comment:
This is an instance where you need a `.` at the end given you are using two sentences. I suggest collapsing this into one. `Name of the individual or company that maintains the operator`. Do you really need to call out that it should not be an email?

Reply:
Good point.